### DISCUSSION TITLE
Keep PhotoPrism folders out of "/photoprism/originals"

### DISCUSSION THREAD

**Discussion by joinso** [photoprism/photoprism#3524] [Q&A] [answered]
I setup PhotoPrism using Docker, and I setup this volumes:
/docker/volumes/photoprism/storage  ---> /photoprism/storage
/photo  ---> /photoprism/originals

In "/photo" are all the folders of my photos like this:
2023
+----01
         +---01
         ....
         +---31
+----02
+----03
After running PhotoPrism I see that PhotoPrism create this folders inside "/photo":

small2x
thumb
thumb2x
tracks
import
medium
medium2x
original
small
There is a way to move this folder to another location like ".photoprism/folders..." ?

Regards,
Jordi
JOINSO

**Comment by lastzero** [comment]
We are currently on vacation, so just a link to our docs:
- https://docs.photoprism.app/getting-started/config-options/#storage

**Comment by joinso** [comment]
First at all, enjoy your holidays.
But the docs does not talk about this folders....

**Reply by lastzero** [reply]
You can configure all storage locations with config options and additionally in your docker-compose.yml if you use Docker.

**Comment by joinso** [comment]
I am using Docker.
I setup only this volumes:
/docker/volumes/photoprism/storage ---> /photoprism/storage
/photo ---> /photoprism/originals
But I do not see in the docs, how to setup the location of the folder that I mention.
For example, I search "medium2x" in the docs and i get no results.

Regards,
Jordi
JOINSO

**Comment by joinso** [comment, marked as answer]
Hi!
We can close this.
Work as expected.
The folders that I mention was generated with other docker container (lychee).

Regards,
Jordi
JOINSO
